### ISSUE TITLE
Bad Allocation Error When Editing An Account While In The Account

### ISSUE THREAD

**Issue by Ocirasai** [moneymanagerex/moneymanagerex#6159]
MMEX version:
 - [X] 1.6.4
 - [ ] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [X] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

I periodically see a Bad Allocation error dialog box show up if I try to edit an account while in that account.

# Reproduction

Is the bug reproducible? 
 - [ ] Always 
 - [X] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Create a new account.
2. Click on the account from the list of accounts on the Dashboard (so that you are now in the account).
3. From the Accounts menu, select Edit Account.
4. Make a change to that account, such as changing the account title.

<img width="302" alt="Bad Allocation" src="https://github.com/moneymanagerex/moneymanagerex/assets/140843655/7bea97df-0553-478d-b4a2-161699e8b667">

Expected result:

1. Make the change as expected or
2. The option to edit the account should be greyed out in the list that is given when Edit Account is selected or
3. No account edits can be made unless initiated from the Dashboard (I.e. the menu entry to Edit Account is greyed out when a user is in an account).

Actual result:

Note: I am seeing this error most of the time when I follow these steps, but not every time (it appeared to just work once or twice).

**Comment by tactilis** [comment]
Thanks for reporting.

"Bad allocation" has been seen when editing the account _and_ at other times.

Duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/5186. Therefore could you close your report please.

 <br>

> Expected result:
>
>1.  Make the change as expected or
>2.  The option to edit the account should be greyed out in the list that is given when Edit Account is selected or
>3.  No account edits can be made unless initiated from the Dashboard (I.e. the menu entry to Edit Account is greyed out when a user is in an account).

Option 1 is the right answer. 2 and 3 are not the way to go.
